tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -2335,7 +2355,28 @@ def test_backfill_seer_grouping_records_cohort_creation_multiple_batches(
     def test_backfill_for_project_with_only_stacktrace(self, mock_post_bulk_grouping_records):
         project = self.create_project(organization=self.organization)
         data = {
-            "stacktrace": {"frames": [{"function": "foo", "filename": "file.php", "in_app": True}]},
+            **ONLY_STACKTRACE,
+            "timestamp": before_now(seconds=10).isoformat(),
+        }
+        event = self.store_event(data=data, project_id=project.id)
+        event.group.times_seen = 2
+        event.group.save()
+        mock_post_bulk_grouping_records.return_value = {"success": True, "groups_with_neighbor": {}}
+        with TaskRunner():
+            backfill_seer_grouping_records_for_project(project.id, None)
+
+        group = Group.objects.get(id=event.group.id)
+        assert group.data["metadata"].get("seer_similarity") == {
+            "similarity_model_version": SEER_SIMILARITY_MODEL_VERSION,
+            "request_hash": ANY,
+        }
+
+    @with_feature("projects:similarity-embeddings-backfill")
+    @patch("sentry.tasks.embeddings_grouping.utils.post_bulk_grouping_records")
+    def test_backfill_for_project_with_threads_stacktrace(self, mock_post_bulk_grouping_records):
```

Comment:
I'm duplicating the test above but using `EVENT_WITH_THREADS_STACKTRACE` instead of `ONLY_STACKTRACE`.